Assets/Mirror/Weaver/Weaver.cs

```diff
@@ -179,7 +179,6 @@ class Weaver
         public static MethodReference sendEventInternal;
         #endregion
 
-        public static ModuleDefinition corLib;
```

Comment:
I am missing something,  where are you declaring CorLibModule?

Reply:
307612c